store/onboardingStore.ts

```diff
@@ -0,0 +1,44 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { create } from 'zustand'
+import { createJSONStorage, persist } from 'zustand/middleware'
+
+type OnboardingState = {
+  selectedMovies: number[]
```

Comment:
we don't have a movies selection in the onboarding, only genres, the first step is just a placeholder images